packages/generator-rest/test/poc/promotion.service.ts

```diff
@@ -0,0 +1,29 @@
+import { AxiosRequestConfig } from 'axios';
+import { buildAxiosRequestConfig } from '@sap-cloud-sdk/core';
+import { CalculationApi, Percent } from './generated/calculation';
+import { transaction } from './promotion-data';
+
+export async function getPromotion(): Promise<Percent> {
+  const token = 'ask the team';
```

Comment:
I don't really get why this is in the promotion-service. But I guess this is just a test for now right...